lib/ruumba/analyzer.rb

```diff
@@ -33,8 +33,8 @@ def run(dir = ARGV)
     # @return [String] The extracted Ruby code.
     def extract(filename)
       File.read(filename).scan(ERB_REGEX).map(&:last)
-        .reject { |line| line[0] == '#' }
-        .map(&:strip).join("\n")
+          .reject { |line| line[0] == '#' }
```

Comment:
Not sure why RuboCop bumped the indentation here

Reply:
probably to align with `.read`